docs/conf.py

```diff
@@ -594,7 +594,7 @@ def _get_params(root_schema: dict, prefix: str = "", default_section: str = "")
 
 # A list of patterns to ignore when finding files
 autoapi_ignore = [
-    'airflow/configuration/',
```

Comment:
This folder doesn't exist anymore, and without the leading `*` it never matched anything anyway.